docs/pieces.md

```diff
@@ -346,7 +341,7 @@ modules: {
 }
 ```
 
-```
+```js
```

Comment:
Isn't it `javascript`?

Reply:
Both are valid and js is more succinct so I'm on board.

Reply:
That works.